designs/manager_policy_mapping/policy_mapping.md

```diff
@@ -0,0 +1,76 @@
+# Policy mapping workflow with IP/Mac
+
+This document describes the proposed changes for policy mapping workflow with IP/Mac.
+
+## Background
+
+Currently, we have two ways to determine which policy gets applied to which workload:
+
+- Provide manager with encapsulation tag
+- Provide manager with IP/Mac
+
+If the manager is supporting policy mapping and security group, we provide manager security group information to bind the policy.
```

Comment:
Mention the name of the properties here referred to in the plugin documentation and/or provide a link to the line of code.  
We provide manager security group -> we provide the manager with the security group  name and members (their IPs and MACs)